docs/docker-postgres-backups.rst

```diff
@@ -1,13 +1,14 @@
 PostgreSQL Backups with Docker
 ==============================
 
-Prerequisites:
+.. note:: For brevity it is assumed that you will be running the below commands against local environment, however, this is by no means mandatory so feel free switching to ``production.yml`` when needed.
```

Comment:
I'm no native English speaker, but isn't it more English to say "feel free to switch"?

Reply:
Me neither, but yours sounds right to me (and [not only me](https://dictionary.cambridge.org/us/dictionary/english/feel-free-to-do-something).)